help/move-to-zulip-cloud.md

```diff
@@ -0,0 +1,135 @@
+# Move from self-hosting to Zulip Cloud
+
+With Zulip's high quality import and export tools, you can always
+move from self-hosting your own Zulip server to using the Zulip
+Cloud service (and back).
+
+## Process overview
+
+To move your Zulip organization from a self-hosted server to Zulip
+Cloud, you will need to take the following steps, which are described
+in more detail below:
+
+{start_tabs}
+
+1. [Plan the process and coordinate with Zulip support](#plan-the-process-and-coordinate-with-zulip-support).
+
+1. [Upgrade your self-hosted server](#upgrade-your-self-hosted-server).
+
+1. [Export organization data](#export-organization-data).
+
+1. [Decide how users will log in](#decide-how-users-will-log-in).
+
+{end_tabs}
+
+## Plan the process and coordinate with Zulip support
+
+To import your self-hosted organization into Zulip Cloud, your server will need
+to have the same database version as Zulip Cloud. Zulip Cloud is updated every
```

Comment:
`database format` is better here -- postgres version isn't important, it's what tables exist.